packages/@okta/i18n/src/properties/login.properties

```
@@ -932,10 +932,11 @@ oie.phone.invalid = Invalid Phone Number.
 oie.email.label = Email
 oie.email.authenticator.description = Verify with a link or code sent to your email
 oie.email.mfa.title = Verify with your email
+oie.email.challenge.mfa.title = Get a verification email
 oie.email.verify.primaryButton = Send me an email
 # {0} is a user's email address
-oie.email.verify.subtitleWithEmailAddress = Verify with an email link or enter a code sent to <$1>{0}</$1>.
-oie.email.verify.subtitleWithoutEmailAddress = Verify with an email link or enter a code sent to your email.
+oie.email.verify.subtitleWithEmailAddress = Send a verification email to <$1>{0}</$1> by clicking on “Send me an email”.
```

```suggestion
oie.email.verify.subtitleWithEmailAddress = Send a verification email to <$1>{0}</$1> by clicking on "Send me an email".
```
Your editor used snazzy smart quotes, but they should be plain ascii quotes.